R/stan_betareg.fit.R

```diff
@@ -110,14 +109,28 @@ stan_betareg.fit <- function(x, y, z = NULL,
   for (i in names(prior_intercept_stuff_z))
     assign(paste0(i, "_z"), prior_intercept_stuff_z[[i]])
 
+  prior_dispersion_stuff <-
+    handle_glm_prior(
+      prior_dispersion,
+      nvars = 1,
+      default_scale = 5,
+      link = NULL, # don't need to adjust scale based on logit vs probit
+      ok_dists = ok_dispersion_dists
+    )
+  # prior_{dist, mean, scale, df, dist_name, autoscale}_for_dispersion
+  names(prior_dispersion_stuff) <- paste0(names(prior_dispersion_stuff), "_for_dispersion")
+  for (i in names(prior_dispersion_stuff)) 
+    assign(i, prior_dispersion_stuff[[i]])
+  
   if (nvars_z == 0) {
       prior_mean_z <- double()
       prior_scale_z <- double()
       prior_df_z <- integer()
   }
   
   # prior scaling (using sd of predictors)
-  if (scaled && prior_dist > 0L && !QR) {
+  min_prior_scale <- 1e-12 # used to be set in prior_options()
```

Comment:
can you remove the comment about `prior_options()`? That really only applies for `stan_glm.fit` because `stan_betareg.fit` hasn't been included in a release yet